pkg/util/push/otel.go

```diff
@@ -132,7 +133,7 @@ func otelMetricsToTimeseries(ctx context.Context, discardedDueToOtelParseError *
 			return nil, err
 		}
 
-		dropped := md.DataPointCount() - sampleCountInMap(tsMap)
+		dropped := len(multierr.Errors(errs))
```

Comment:
Isn't this going to panic (due to conversion failure) if there is only one error?

Reply:
Nope. If you look at the code, it returns a slice with the original error if type assertion fails.

Reply:
See: https://github.com/uber-go/multierr/blob/v1.8.0/error.go#L195

Reply:
You're right. I misread `multierr.Errors(...)` as a type assertion, but it's actually function that returns slice of errors.